### ISSUE TITLE
Notify driver when ride is cancelled

### ISSUE THREAD

**Issue by AlexGilleran** [RHoKAustralia/hills-carpal#77]
Note that this depends on #76 so that we can actually send the email!

When a ride is cancelled by a facilitator, if a driver has already accepted it then that driver needs to be notified, otherwise they won't know that they're not supposed to turn up for the ride!

In order to achieve this, we need to make it so that when ride is cancelled by a facilitator, _and_ it's already been claimed by a driver, the driver automatically gets an email to let them know.

The email should be something like:

```
Subject: Ride for {client_name} has been cancelled

Hi {driver_name},

This email is to let you know that  that the ride you accepted for {client_name} from {from_address} to {to_address} is no longer needed and has been cancelled. If you'd like to give a ride to someone else, <a href={link to find a ride}>look to see if there's any rides needed</a>.

Thanks,
Hills Carpal.
```

Because we record the driver's email directly against a ride right now, we should be able to get this done without any of the user synchronisation stuff in #78, although if it's done before #78 it might need a bit of an update to use whatever new database structure comes about as a result of completing that issue.

# Criteria
- When a facilitator cancels a ride:
    - If the ride has already been accepted by a driver, that driver gets the email above
    - If the ride has not been accepted by anyone, nothing additional happens
